file-system-lesson-11/main.js

```diff
@@ -0,0 +1,87 @@
+const fs = require('fs');
```

Comment:
Do not use "require". Change file type from .js to .mjs and use esm modules
```javascript
import fs from 'node:fs';
``` 
When you import node core packages use 'node:package-name' instead 'package-name'